src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -245,29 +243,20 @@
     BitwiseXorSegment=StringParser(
         "^", SymbolSegment, name="binary_xor", type="binary_operator"
     ),
-    EqualsSegment=StringParser(
-        "=", SymbolSegment, name="equals", type="comparison_operator"
-    ),
     LikeOperatorSegment=NamedParser(
         "like_operator", SymbolSegment, name="like_operator", type="comparison_operator"
     ),
-    GreaterThanSegment=StringParser(
-        ">", SymbolSegment, name="greater_than", type="comparison_operator"
-    ),
-    LessThanSegment=StringParser(
-        "<", SymbolSegment, name="less_than", type="comparison_operator"
-    ),
-    GreaterThanOrEqualToSegment=StringParser(
-        ">=", SymbolSegment, name="greater_than_equal_to", type="comparison_operator"
+    RawNotSegment=StringParser(
+        "!", SymbolSegment, name="raw_not", type="raw_comparison_operator"
     ),
-    LessThanOrEqualToSegment=StringParser(
-        "<=", SymbolSegment, name="less_than_equal_to", type="comparison_operator"
+    RawEqualsSegment=StringParser(
+        "=", SymbolSegment, name="raw_equals", type="raw_comparison_operator"
```

Comment:
@barrywhart I've added the raw segments as we discussed [here](https://github.com/sqlfluff/sqlfluff/pull/2197#discussion_r775085427). Now `.is_type("comparison_operator")` will unambiguosly refer to the combined comparison operators.